bot/exts/info/python_news.py

```diff
@@ -72,6 +73,12 @@ async def get_webhook_names(self) -> None:
             if mail["name"].split("@")[0] in constants.PythonNews.mail_lists:
                 self.webhook_names[mail["name"].split("@")[0]] = mail["display_name"]
 
+    @staticmethod
+    def escape_markdown(content: str) -> str:
+        """Escape the markdown underlines."""
+        # taken from discord.utils.escape_markdown
+        return re.sub(r'[_\\~|]', lambda match: '\\' + match[0], content, 0, re.MULTILINE)
```

Comment:
If this is copied from `discord.utils.escape_markdown`, why aren't we using it?

Reply:
Actually I removed most of the code it's not really the same function anymore. We don't because apparently we want to keep some markdown
https://discord.com/channels/267624335836053506/635950537262759947/873320120494211152

Reply:
I should remove this comment